.pylintrc

```diff
@@ -17,6 +17,23 @@ disable=bad-continuation,
         missing-return-type-doc,
         wrong-import-order,
 
+
+good-names=
+    # PyLint's default good names.
+    i,j,k,ex,Run,_,ok,
+
+    # The standard name used for the pyramid_tm transaction manager.
+    tm,
+
+    #websocket
+    ws,
+    #request.db
+    db,
+    #request.es
+    es,
+    # Primary key on models
+    id,
```

Comment:
Does this prevent Pylint warning about attributes as well as variables?

_Very_ not sure about this one. We shouldn't be shadowing the built in `id` here? And this would seem to allow it.

Reply:
This makes it a good name as long as it not shadowing the builtin (so just on class attributes?)

just tried it and I still get: `h/views/main.py:94:4: W0622: Redefining built-in 'id' (redefined-builtin)`